src/core/proj/qgscoordinatereferencesystem.cpp

```diff
@@ -2616,6 +2617,18 @@ int QgsCoordinateReferenceSystem::syncDatabase()
         proj4 = "";
       }
 
+      // there's a not-null contraint on these columns, so we must use empty strings instead
+      QString operation = "";
+      QString ellps = "";
+      getOperationAndEllipsoidFromProjString( proj4, operation, ellps );
+
+      const QString translatedOperation = QgsCoordinateReferenceSystemUtils::translateProjection( operation );
+      if ( translatedOperation.isEmpty() && !operation.isEmpty() )
+      {
+        std::cout << QStringLiteral( "Operation needs translation in QgsCoordinateReferenceSystemUtils::translateProjection: %1" ).arg( operation ).toLocal8Bit().constData() << std::endl;
```

Comment:
Deliberately not using QgsDebugMsg here?

Reply:
Yep -- it won't show up in the cli output from crssync.

Reply:
@nyalldawson , roger that.